tools/travis/runStandaloneTests.sh

```diff
@@ -44,6 +44,8 @@ kind create cluster --wait 5m
 export KUBECONFIG="$(kind get kubeconfig-path)"
 kubectl config set-context --current --namespace=default
 
+docker pull openwhisk/action-nodejs-v10:nightly
+docker pull openwhisk/dockerskeleton:nightly
```

Comment:
seems travis ci machine will take a long time to pull images which make some tests failed